DSharpPlus/Logging/LoggerEvents.cs

```diff
@@ -132,9 +132,13 @@ public static class LoggerEvents
         /// </summary>
         public static EventId RestTx { get; } = new EventId(124, "REST ↑");
 
+        public static EventId RestCleaner { get; } = new EventId(125, nameof(RestCleaner));
+
+        public static EventId RestHashMover { get; } = new EventId(126, nameof(RestHashMover));
+
         /// <summary>
         /// Events pertaining to Discord API requests from the <see cref="DiscordShardedClient"/>.
         /// </summary>
-        public static EventId ShardRest { get; } = new EventId(125, nameof(ShardRest));
+        public static EventId ShardRest { get; } = new EventId(127, nameof(ShardRest));
```

Comment:
In the future, events should be appended, as moving their IDs is considered a breaking change.